.github/workflows/krel-release-notes-validate.yaml

```diff
@@ -78,16 +78,6 @@ jobs:
           KREL_PATH="$HOME/.local/bin/krel"
           echo "krel-path=$KREL_PATH" >> "$GITHUB_OUTPUT"
           echo "Krel installed at: $KREL_PATH"
-      - name: Get the latest commit from the base branch
-        id: base_branch_commit
-        run: |
-          echo "commit=$(git rev-parse origin/master)" >> $GITHUB_OUTPUT
-      - name: Get all changed release files
-        id: changed-files
-        uses: tj-actions/changed-files@cc733854b1f224978ef800d29e4709d5ee2883e4 # v44.5.5
-        with:
-          base_sha: ${{ steps.base_branch_commit.outputs.commit }}
-          files: releases/**/*.yaml
```

Comment:
@npolshakova I removed this section. `changed-files` are being calculated in the next step (using bash and git) and this wasn't being used anywhere.

Reply:
Perfect!